src/resources/elements/lazylinkfield.js

```diff
@@ -55,6 +56,10 @@ export class LazyLinkfield extends Field {
         target[lastFieldPathEntry] = value;
       }
     }
+    if (this.cachedValue) {
```

Comment:
Get in the habit of describing your code with comments. Plain language comments help reviewers and collaborators to understand what the code is intended to do.